src/_includes/docs/install/reqs/macos/zsh-config.md

```diff
@@ -0,0 +1,51 @@
+{% assign target = include.target %}
+
+<details markdown="1">
+<summary><strong>To verify your shell configuration, expand this section</strong></summary>
+
+Like most UNIX-like operating system, macOS can support multiple shells,
+like `bash`, `zsh`, and `sh`.
+
+As of the October 2019 release of macOS Catalina (macOS 10.15),
+Z shell or `zsh` is the default shell for macOS.
+Apple made this change to due to licensing changes for `bash`.
+
+#### Check and set `zsh` as default
+{:.no_toc}
+
+1. To verify `zsh` has been installed and set as
+   the default macOS shell, run the following commands.
+
+    ```terminal
+    $ which zsh; dscl . -read ~/ UserShell
+    ```
+
+    {{terminal}} should print the following as its response.
+
+    ```terminal
+    /bin/zsh
+    UserShell: /bin/zsh
+    ```
+
+1. If you need to install `zsh`, run the following command.
+
+    ```terminal
+    $ brew install zsh
+    ```
+
+   To check if your install succeeded,
+   run the command in the previous step after installing `zsh`.
+
+1. If you need to change your default shell, run the following command.
```

Comment:
We should explain what the command does before we tell them to run it:

```suggestion
1. To change your shell to Zsh, run the following command.
```

Reply:
I might need to tweak that slightly. This step is optional.